templates/cli/lib/commands/command.js.twig

```diff
@@ -37,13 +40,27 @@ const {{ service.name | caseLower }}{{ method.name | caseUcfirst }} = async ({ {
     if (!fs.lstatSync(folderPath).isDirectory())
         throw new Error('The path is not a directory.');
     
+    const ignorer = ignore();
+
+    const xfunction = localConfig.getFunction(functionId);
+
+    if(xfunction.ignore) {
+        ignorer.add(xfunction.ignore);
+        log('Ignoring files using configuration from appwrite.json');
+    } else if(fs.existsSync(pathLib.join({{ parameter.name | caseCamel | escapeKeyword }}, '.gitignore'))) {
+        ignorer.add(fs.readFileSync(pathLib.join({{ parameter.name | caseCamel | escapeKeyword }}, '.gitignore')).toString());
+        log('Ignoring files using configuration from .gitignore');
```

Comment:
```suggestion
        log('Ignoring files in .gitignore');
```

Reply:
Also updated other message to `Ignoring files in appwrite.json`